bve-render/src/render.rs

```diff
@@ -203,6 +208,10 @@ pub fn create_depth_buffer(device: &Device, size: PhysicalSize<u32>, samples: MS
 }
 
 pub fn create_framebuffer(device: &Device, size: PhysicalSize<u32>, samples: MSAASetting) -> TextureView {
+    debug!(
```

Comment:
This doesn't matter yet, but you probably will want to name your framebuffers and log their names here

Reply:
I mean I only got one.

Reply:
For now, but once you get more complex you may have lots of framebuffers. Just a thought

Reply:
Yeah I should be specific with my non-main framebuffer